rust-symcrypt/BUILD.md

```diff
@@ -26,18 +28,18 @@ You can configure your system one of 3 ways.
     ```
     **Note:** This change will only persist within the current process, and you must re-set the PATH environment variable after closing the PowerShell window.
 
-2. Manually copy the `symcrypttestmodule.lib` to `C:\Windows\System32`
-    Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypttestmodule.lib`
+2. Manually copy the `symcrypt.lib` to `C:\Windows\System32`
+    Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypt.lib`

```

Comment:
I wouldn't recommend this. It's not really good practice to drop development dependencies into System32.

Reply:
This is what are currently recommending for users that consume our product, will follow up offline